x-pack/plugins/cases/public/components/create/form.tsx

```diff
@@ -86,9 +86,8 @@ export const CreateCaseFormFields: React.FC<CreateCaseFormFieldsProps> = React.m
   ({ connectors, isLoadingConnectors, withSteps, owner, draftStorageKey }) => {
     const { isSubmitting } = useFormContext();
     const { isSyncAlertsEnabled, caseAssignmentAuthorized } = useCasesFeatures();
-
     const availableOwners = useAvailableCasesOwners();
-    const canShowCaseSolutionSelection = !owner.length && availableOwners.length;
+    const canShowCaseSolutionSelection = !owner.length && availableOwners.length > 1;
```

Comment:
I think this will slightly change the behavior of the form when going through Analytics in Classic Kibana.

There there is no `owner` and the `availableOwners` can be 1(depending on the role's permissions). This will make it not show the owner picker(your PR) vs showing the owner picker with all options disabled except one(main).

It is a specific scenario so maybe it is not a problem.

Reply:
hmm..The only way to handle this could be to add another config variable which tells weather it is serverless or classic kibana and then put the check accordingly.  

However I think it's `okay to not show owner selector picker when only one owner` because we don't show it anyway when it's a security solution dashboard and user has permission for cases in security in classic kibana.

Reply:
Yeah but there we don't show because we are in Security Solution. The owner is defined and it is safe to assume the created case will end up inside Security Solution.

In the scenario I mentioned, we start from `Analytics`. The user has no indication of where the case will end up.

(Although he can assume if he only has access to like, Observability 😄 )

I think this is minor though. Maybe double-check with @mdefazio ?

Reply:
I'm good with hiding this if there is only one solution available.  Thanks for double checking on this.  @shanisagiv1 Let me know if you're ok with hiding this solution selector (and if you would like some more context to this, happy to zoom 😉  )